### DISCUSSION TITLE
SponsorBlock - Auto-skip - less temperamental?

### DISCUSSION THREAD

**Discussion by felichas** [FreeTubeApp/FreeTube#6175] [General]
With v0.22.0 Beta I seem to have lost a "feature?" I found myself using from time to time: being able to watch skipped SponsorBlock segments at will.

SponsorBlock would skip a block in the middle of a video, and I decided I wanted to watch it, I could just jump back a couple seconds and watch it. Kind of "if the player is at the beginning of the block, the block will be skipped. If the player is in the middle of the block, the block will not be skipped".

This behaviour worked very well for me, but it is gone now as it will always skip the sponsor block, no matter what.

I searched for it and seems to be the result of [fixing a bug](https://github.com/FreeTubeApp/FreeTube/discussions/5948#discussioncomment-11072058).
Any chance this previous helpful behaviour can return while keeping the bug fixed?

**Comment by absidue** [comment]
To quote myself in #4978:
> I removed the ability to seek into sponsor block segments that are set to skip, as it was causing the automatic skipping to not work properly, due to the existing implementation being fundamentally flawed. If someone would like to add something like that back in the future, it should require a specific user-interaction to unskip the segment, e.g. a button in the skip message, instead of allowing users to manually seek into the segment.
